packit/cli/source_git_status.py

```diff
@@ -33,7 +34,7 @@ def source_git_status(config: Config, source_git: str, dist_git: str):
     source_git_path = pathlib.Path(source_git).resolve()
     dist_git_path = pathlib.Path(dist_git).resolve()
     package_config = get_local_package_config(
-        source_git_path, package_config_path=config.package_config_path
+        package_config_path=str(source_git_path / DISTRO_DIR / SRC_GIT_CONFIG)
```

Comment:
Can we change the `package_config_path` parameter of `get_local_package_config()` to accept `Path` as well, i.e. from `Optional[str]` to `Optional[Union[Path, str]]`?